src/vorta/views/archive_tab.py

```diff
@@ -297,8 +320,71 @@ def receive():
                         else:
                             self._set_status(params['message'])
 
-                dialog = choose_file_dialog(self, "Choose Extraction Point")
+                dialog = choose_file_dialog(self, "Choose Extraction Point", want_folder=True)
                 dialog.open(receive)
 
     def extract_archive_result(self, result):
         self._toggle_all_buttons(True)
+
+    def update_mount_button_text(self):
+        snapshot_name = self.selected_archive_name()
+        if not snapshot_name:
+            return
+
+        if snapshot_name in self.mount_points:
+            self.set_mount_button_mode('Unmount')
+        else:
+            self.set_mount_button_mode('Mount')
+
+    def open_folder_action(self):
+        snapshot_name = self.selected_archive_name()
+        if not snapshot_name:
+            return
+
+        try:
+            mount_point = self.mount_points[snapshot_name]
+        except KeyError:
+            mount_point = None
+
+        if mount_point:
+            open_folder(mount_point)
```

Comment:
above you used "is not None", maybe always use the same method.

if the empty string or any other falsy value except None can not happen, the shorter one is nicer.

Reply:
I tend to use the implicit check in my code but I noticed Vorta sources tend to prefer the "is not None" kind of check, so I tried to use it everywhere to keep the code style but this one seem to have slipped. I'll fix it. Thanks for the review!